unlock-app/src/components/interface/checkout/main/Returning.tsx

```diff
@@ -191,9 +191,15 @@ export function Returning({
                 Sign message
               </Button>
             ) : (
-              <div className="flex justify-between gap-4">
+              <div
+                className={`gap-4 ${
+                  paywallConfig?.endCta
+                    ? 'grid grid-cols-1'
+                    : 'flex justify-between '
+                }`}
+              >
```

Comment:
As the text can be very long when a custom CTA in present lets show buttons in 2 rows

<img width="500" alt="Screenshot 2023-04-06 at 12 29 53" src="https://user-images.githubusercontent.com/20865711/230351593-9005b6da-a3dd-4af8-9d51-86f1e00c204e.png">